### PR TITLE
E2E test cases and GH action implementation

### PR DESCRIPTION
**Pull request by alvitazwar**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# e2e test for Wp-menu-custom-fields
+
+name: CI for rtsocial Plugin Test cases
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [develop]
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:9]: @alvitazwar 
The branches should be develop and master similar to https://github.com/rtCamp/wp-menu-custom-fields/blob/594beed7511097aa2d58b95a5701b261348ef6fe/.github/workflows/e2e.yml#L9

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  pull_request:
+    branches: [develop]
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:11]: same here as well https://github.com/rtCamp/wp-menu-custom-fields/blob/594beed7511097aa2d58b95a5701b261348ef6fe/.github/workflows/e2e.yml#L9

**Review by pooja-muchandikar — COMMENTED** [review]
@alvitazwar Remove .DS_Store file which is under - `tests/.DS_Store`

Remove Artifacts folder its not necessary

**Comment by alvitazwar** [comment]
Closing this PR due to some File conflicts  will create a PR

**Pull request closed without merging**